src/driver/drv_bl_shared.c

```diff
@@ -32,25 +63,26 @@ struct {
 	// It even fails to publish with -1 error (can't alloc next packet)
 	// So we publish when value changes from certain threshold or when a certain time passes.
 	float changeSendThreshold;
-	double lastReading; //double only needed for energycounter i.e. OBK_CONSUMPTION_TOTAL to avoid rounding errors as value becomes high
+	double *lastReading; //double only needed for energycounter i.e. ES_CONSUMPTION_TOTAL to avoid rounding errors as value becomes high
+						 //This is a pointer to `readings` variable, so value accessible via DRV_GetReading()
```

Comment:
Incorrect spelling. "This could be a pointer to `readings` variable". Not mandatory requirement.